ntbs-service/Helpers/RouteHelper.cs

```diff
@@ -16,6 +17,12 @@ public static string GetNotificationPath(string subPath, int id, bool isBeingSub
 
             return string.Format(NotificationBasePath, id, path);
         }
+
+        public static string GetAlertPath(string subPath, int id)

```

Comment:
What do you think about swapping the order of the parameters here, so it's more in line with what it produces?